lib/http2.c

```diff
@@ -1712,6 +1722,7 @@ static ssize_t http2_recv(struct Curl_easy *data, int sockindex,
 
     stream->pausedata += nread;
     stream->pauselen -= nread;
+    drain_this(data, httpc);
```

Comment:
I cannot convince myself this is correct or not, I couldn't understand how drains work. But from the usage of other are of the file, it seems fine.